assignment.js

```diff
@@ -64,9 +66,17 @@ Use the following test cases to confirm your program meets the success criteria
 
 console.log('Problem 2:')
 
-// Add your code below this line
+const values = [3, 48, 29, 12, 18]
+let maxValue = 0
 
-// Add your code above this line
+for(arrayIndex = 0; arrayIndex <= values.length; arrayIndex++){
+
+  if(values[arrayIndex] > maxValue){
+    maxValue = values[arrayIndex]
+    }
+  }
+
+console.log(maxValue)
 
```

Comment:
@nmikhol Good job here implementing the for loop and using the if statement to check for the max value. 
One suggestion is when you are using `values.length` the value you get is `5` so based on the current code, your for loop will iterate one more time than it has to so you may want to use `arrayIndex < values.length`. Take a look at an alternate this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L77)